tests/sentry/api/endpoints/test_organization_member_team_details.py

```diff
@@ -581,7 +583,7 @@ def test_manager_can_remove_members(self):
     def test_manager_can_remove_members_using_user_token(self):
         self.login_as(self.manager)
 
-        scopes = ["org:write", "team:write"]
+        scopes = ["org:write", "team:admin"]
```

Comment:
should we check if this works using _only_ the `team:admin` scope?

Reply:
The test checks this because its looping over each scope individually